packages/concerto-core/lib/decoratormanager.js

```diff
@@ -438,6 +438,8 @@ class DecoratorManager {
                     if (this.falsyOrEqual(target.type, declaration.value.$class)) {
                         this.applyDecorator(declaration.value, type, decorator);
                     }
+                } else {
+                    this.applyDecorator(declaration, type, decorator);
```

Comment:
Applying decorator command sets to map declarations. Without it decorator sets were not getting applied to map declarations

Reply:
@sanketshevkar are you sure you are using the command set correctly, as per the [spec](https://github.com/accordproject/concerto/pull/722)? The branch on line 422 handles the case for Map declarations. Is this a bug?

Reply:
Yes, without the about fix, we cannot apply decorator command-set to a map declaration. I was able to add decorator command-set to key and value by following the spec you shared.
But I found a bug wherein I was not able to apply decorator command-set to a map declaration.

Reply:
`cannot apply decorator command-set to a map declaration` - can you provide an example instance of the command set which repros this scenario?   

There a number of ways in which you might apply a command-set to a map. It sounds as though you would like to decorate the top level of the declaration, and not the type elements themselves? ie something like:

```
@Foo 
map Thing {
 o String
 o String
}
```

Reply:
Yep like this. Might even have cases wherein the key and value elements will need decoration.
```
@Term('Thing')
map Thing {
@Term('uuid')
 o String
@Term('Thing info')
 o String
}
```

Reply:
> I'm open to correction here, but I believe the intention of command-sets is to decorate properties on a Concept. In the case of a Map Declaration, the intention is to decorate its types.

I can confidently talk about vocabulary. In case of vocabulary, it is applied to both the declaration and its properties.
https://replit.com/@dselman/AccordProjectConcerto-Vocabulary
The `@Term('some text')` or `@Term_description('some text')` can be applied to both declaration and properties. 

For applying vocabulary generated decorated sets the following can be applied to declarations:
```
{
    "$class" : "org.accordproject.decoratorcommands@0.3.0.DecoratorCommandSet",
    "name" : "web",
    "version": "1.0.0",
    "commands" : [
        {
            "$class" : "org.accordproject.decoratorcommands@0.3.0.Command",
            "type" : "UPSERT",
            "target" : {
                "$class" : "org.accordproject.decoratorcommands@0.3.0.CommandTarget",
                "namespace" : "test@1.0.0",
                "declaration" : "Thing"
            },
            "decorator" : {
                "$class" : "concerto.metamodel@1.0.0.Decorator",
                "name" : "Term",
                "arguments" : ['Some information about thing']
            }
        }
    ]
}
```
Would result in

```
@Term('Some information about thing')
map Thing {
  o String
  o String
}
```

Reply:
Got it. So the branch handles the case where top level was not annotated. Can you add test coverage for that too please?

Reply:
Addressed.